src/Components/List/List.jsx

```diff
@@ -37,21 +37,55 @@ export default function List({ items, token }) {
     history.push('/add-item');
   };
 
+  const [searchItem, setSearchItem] = React.useState('');
+  const handleChange = (e) => {
+    setSearchItem(e.target.value);
+  };
+
+  const resetSearch = () => {
+    setSearchItem('');
+  };
+
+  const results = !searchItem
+    ? items
+    : items.filter((item) =>
+        item.name.toLowerCase().includes(searchItem.toLocaleLowerCase()),
```

Comment:
I've never used `toLocaleLowerCase()` before, but this is a great idea to account for other languages!